environments/demo/demo-platform-hmcts-net.yml

```diff
@@ -20,4 +20,22 @@ cname:
     record: "hmcts-demo.azurefd.net"
   - name: "afdverify.reformscan"
     ttl: 60
-    record: "afdverify.hmcts-demo.azurefd.net"
\ No newline at end of file
+    record: "afdverify.hmcts-demo.azurefd.net"
+  - name: "_c90f6fda2be96a09fcc613bc71ce799f.moneyclaims"
+    ttl: 60
+    record: "86FB2AAB19A8F5E5C8E7A30814BD0277.E99F825ACD0F3A60C735A1826AE278C9.cb41662fe265cad3ff1b.comodoca.com."
+  - name: "_c90f6fda2be96a09fcc613bc71ce799f.moneyclaims-legal"
+    ttl: 60
+    record: "86FB2AAB19A8F5E5C8E7A30814BD0277.E99F825ACD0F3A60C735A1826AE278C9.cb41662fe265cad3ff1b.comodoca.com."
+  - name: "_c90f6fda2be96a09fcc613bc71ce799f.www.moneyclaims"
```

Comment:
have you checked that these are still needed? they shouldn't be in use anymore

Reply:
Thiru has confirmed the team need them renewed